templates/template-primary-secondary.js

```diff
@@ -0,0 +1,59 @@
+import '../components/colors/colors.js';
+import { css, html, LitElement } from 'lit-element/lit-element';
+
+class TemplatePrimarySecondary extends LitElement {
+
+	static get properties() {
+		return {};
+	}
+
+	static get styles() {
+		return css`
+			.container {
+				display: grid;
+				grid-template-columns: 2fr auto 1fr;
+				grid-template-rows: auto auto auto;
+				grid-template-areas:
+				"header header header"
+				"primary divider secondary"
+				"footer footer footer";
+			}

```

Comment:
I think the design implied that the template should always be the full height of the viewport. From the demo screenshot, I'm not sure it's doing that?

Reply:
Yeah, it isn't right now. We discussed this briefly in the meeting about how we will need to calculate the height of the header and footer and subtract that from the viewport height to determine the height of the primary and secondary panels. Haven't gotten to figuring out how to do that yet.

Reply:
There might be other techniques we could use as well that don't involve quite as much constant re-calculating and observing. I can help with that when you get to it.

Reply:
`flex-direction: column` worked out pretty well for both `d2l-card` and `d2l-dialog`.

Reply:
I've fixed it in this commit by setting `height: 100vh` on the container and fixing an error I had in the grid-template-row height. https://github.com/BrightspaceUI/core/pull/312/commits/4bc74b573d7b0c58f22c3d7e7d0de1b860f44b34